mongodbatlas/provider.go

```diff
@@ -187,16 +189,16 @@ func getResourcesMap() map[string]*schema.Resource {
 		"mongodbatlas_federated_settings_org_config":         resourceMongoDBAtlasFederatedSettingsOrganizationConfig(),
 		"mongodbatlas_federated_settings_org_role_mapping":   resourceMongoDBAtlasFederatedSettingsOrganizationRoleMapping(),
 		"mongodbatlas_federated_settings_identity_provider":  resourceMongoDBAtlasFederatedSettingsIdentityProvider(),
+		"mongodbatlas_serverless_instance":                   resourceMongoDBAtlasServerlessInstance(),
 	}
 	return resourcesMap
 }
 
 func addBetaFeatures(provider *schema.Provider) {
 	if ProviderEnableBeta {
-		provider.ResourcesMap["mongodbatlas_serverless_instance"] = resourceMongoDBAtlasServerlessInstance()
-
-		provider.DataSourcesMap["mongodbatlas_serverless_instance"] = dataSourceMongoDBAtlasServerlessInstance()
-		provider.DataSourcesMap["mongodbatlas_serverless_instances"] = dataSourceMongoDBAtlasServerlessInstances()
+		// provider.ResourcesMap["mongodbatlas_example"] = resourceMongoDBAtlasExample)
+		// provider.DataSourcesMap["mongodbatlas_example"] = dataSourceMongoDBAtlasExample()
```

Comment:
[nit] Just remove, rather than commenting out. We can use git history to find later if needed.